use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -9,27 +9,24 @@ cloud_ui:
         - [services, :serviceId, overview]
 ---
 
-# Tier data to the object storage tier
+# Manage tiering
 
-Enable tiered storage to begin migrating rarely used data from Timescale's standard high-performance storage tier
-to the object storage tier to save on storage costs. 
+You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from Timescale's standard high-performance storage to the object storage. With tiered storage enabled, you then either create automated tiering policies or manually tier and untier data.
 
-## Enabling the object storage tier
+Data on the object storage tier cannot be modified - so inserts, updates, and deletes will not work on tiered data. Make sure that you are not tiering data that is being <b>actively modified</b> to the object storage tier.
```

Comment:
Add a not saying this is for the Scale and Enterprise tiers only. Andddddd, please use the tiering widget in the page. Yaya